.devcontainer/Dockerfile

```diff
@@ -5,8 +5,14 @@ RUN apt-get update && apt-get install -y rustc cargo nodejs npm python3-sphinx p
 # Install required Sphinx extensions for the documentation
 RUN pip3 install furo sphinx-inline-tabs
 
+# Install required Sphinx extensions for the documentation
+RUN pip3 install furo sphinx-inline-tabs
+
 # Install just, a task runner
-RUN cargo install --root /usr/local just cargo-release llvm-conv
+RUN cargo install --root /usr/local just cargo-release 
+
+# Install llmv-conv
+RUN cargo install cargo-llvm-cov --locked
```

Comment:
Locked is not necessary here. I had the wrong name, but there is no cargo.toml file so it won't find the version in any case.

Reply:
Removed; blindly following online instructions to get round an error during the build. could probably have moved it back to line 12.

Reply:
In fact, I'll do that now.